x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -525,6 +525,132 @@ export class EndpointDocGenerator extends BaseDataGenerator {
     };
   }
 
+  /**
+   * Creates a host metrics document
+   * @param ts - Timestamp to put in the event
+   * @param hostInfo - Host metadata
+   * @param metadataDataStream the values to populate the data_stream fields when generating metadata documents
+   */
+  public generateHostMetricsData(
+    ts = new Date().getTime(),
+    hostInfo: HostMetadata,
+    metricsDataStream: DataStream
+  ) {
```

Comment:
Can you add a return type here

Reply:
Also - do we have a need to generate this data standalone?

I been trying to limit adding more data to this generator here (`generate_data.ts`) and instead create smaller more single-purpose generators so that they can be consumed individually.

Reply:
I can create a separated script like the artifacts generators if that will maintain this cleaner